hw/ip_templates/racl_ctrl/rtl/racl_ctrl.sv.tpl

```diff
@@ -123,8 +123,15 @@ module ${module_instance_name} import ${module_instance_name}_reg_pkg::*; #(
   assign policy_${policy['name'].lower()}.write_perm = reg2hw.policy_${policy['name'].lower()}${"_shadowed" if enable_shadow_reg else ""}.write_perm.q;
 
 % endfor
+% if max(nr_policies - len(policies), 0) > 0:
+  localparam racl_policy_t UnusedPolicy = '0;
+% endif
+
```

Comment:
Not a big deal, but this gives a double newline with the one from line 124 above. It's easy to fix though: I think you can just swap lines 128 and 129.